provision/acc_provision/apic_provision.py

```diff
@@ -2752,6 +2849,175 @@ def l3out_filter(self):
         self.annotateApicObjects(data)
         return path, data
 
+    def cluster_l3out_cif_global(self):
+        system_id = self.config["aci_config"]["system_id"]
+        tenant = self.config["aci_config"]["cluster_l3out"]["vrf"]["tenant"]
+        vrf_tenant = self.config["aci_config"]["vrf"]["tenant"]
+
+        path = "/api/mo/uni/tn-%s/cif-%s-l3out-allow-all.json" % (tenant, system_id)
+        data = collections.OrderedDict(
+            [
+                (
+                    "vzCPIf",
+                    collections.OrderedDict(
+                        [
+                            (
+                                "attributes",
+                                collections.OrderedDict(
+                                    [
+                                        (
+                                            "name",
+                                            "%s-l3out-allow-all"
```

Comment:
It would be better to name this to xxx-export so it is clear that this is the exported contract